src/main/java/Command/FindCommand.java

```diff
@@ -0,0 +1,34 @@
+package Command;
+
+import Task.Task;
+import Task.TaskList;
+import Dook.Ui;
+import Dook.Storage;
+import java.util.ArrayList;
+
+public class FindCommand extends Command {
+
+    private final String target;
+
+    public FindCommand(String target) {
+        this.target = target;
+    }
+
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
+        ArrayList<Task> found = new ArrayList<>();
```

Comment:
Perhaps it would be good to rename this to make it clear that it is a multi-valued variable.